queue/linked_list.py

```diff
@@ -0,0 +1,64 @@
+class Node:
+    def __init__(self, value=None, next_node=None):
+        self.value = value
+        self.next_node = next_node
+
+    def get_value(self):
+        return self.value
+
+    def get_next(self):
+        return self.next_node
+
+    def set_next(self, new_next):
+        self.next_node = new_next
+
+
+class LinkedList:
+    def __init__(self):
+        self.head = None
+        self.tail = None
+
+    def add_to_tail(self, value):
```

Comment:
code is looking clean but I recommend you document the functions and classes as well. python has something called Docstring. check it out - https://www.python.org/dev/peps/pep-0257/